src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1257,6 +1344,26 @@ core::Error initialize()
    return initBlock.execute();
 }
 
+boost::regex retrieveFileLineNumberLineRegex(bool gitFlag)
```

Comment:
Naming: `retrieve` usually implies "fetch from somewhere" -- would suggest `getFileLine...` or just `fileLine...`

Reply:
Fixed in latest commit